test/voice-broadcast/stores/VoiceBroadcastRecordingsStore-test.ts

```diff
@@ -110,6 +99,31 @@ describe("VoiceBroadcastRecordingsStore", () => {
             it("should emit a current changed event", () => {
                 expect(onCurrentChanged).toHaveBeenCalledWith(null);
             });
+
+            it("and calling it again should work", () => {
+                recordings.clearCurrent();
```

Comment:
This seems to be missing an `expect` statement?

Reply:
It should just not explode :laughing:

Reply:
A comment then please?